reflex/components/radix/primitives/drawer.py

```diff
@@ -51,17 +66,14 @@ class DrawerRoot(DrawerComponent):
     # Duration for which the drawer is not draggable after scrolling content inside of the drawer. Defaults to 500ms
     scroll_lock_timeout: Var[int]
 
-    # When `False`, it allows to interact with elements outside of the drawer without closing it. Defaults to `True`.
-    modal: Var[bool]
-
-    # Direction of the drawer. Defaults to `"bottom"`
-    direction: Var[LiteralDirectionType]
-
     # When `True`, it prevents scroll restoration. Defaults to `True`.
     preventScrollRestoration: Var[bool]
 
-    # Fires when the drawer is opened.
-    on_open_change: EventHandler[identity_event(bool)]
+    # Enable background scaling, it requires an element with [vaul-drawer-wrapper] data attribute to scale its background.
```

Comment:
unclear what this means, but based on https://www.shadcn-vue.com/docs/components/drawer#scale-background, it seems `vaul-drawer-wrapper` isn't a "data attribute", but just a regular attribute, likely applied via `special_props=[Var("vaulDrawerWrapper")]`, but i haven't tested it

```suggestion
    # Enable background scaling, it requires container element with `vaul-drawer-wrapper` attribute to scale its background.
```

Reply:
fixed